.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+    - nodejs_version: "6"
+    - nodejs_version: "7"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - yarn install
```

Comment:
mh? i do not install yarn, the install block will also install the dependencies i think.
i looked at the appveyor-files from fb-projects